node_modules/.bin/cssesc

```diff
@@ -0,0 +1 @@
+../cssesc/bin/cssesc
```

Comment:
The reason you have more than 3000 thousand files on your repo is that you didn't ignore node_modules on your .gitignore file 👀 Please kindly fix this issue 😬